### ISSUE TITLE
Steps autocomplete doesn't work.

### ISSUE THREAD

**Issue by denkorz** [SpecFlowOSS/SpecFlow.VS#27]
Steps autocomplete doesn't work. But it works fine on the same project with VS 2019 and extention for VS 2019.
When I start writing: '_When I co_'... I expect to get a list of existing steps having these words like '_When I configure green window as follows:_' or '_When I save configuration for green window_'

**Comment by patiqs** [comment]
Do you observe the same after rebuilding the solution?

**Comment by denkorz** [comment]
Yes, but I've investigted a bit more and found out that step autocomplete works fine when step beggining was chosen from autocomplete.
So if I write 'When I', nothing happens, but if I write 'When', wait for popup and press Enter, then I can see autocomplete options with existing steps.
BUT, on previous version I could find **real steps with parameters** examples from some other feature files. And with new autocomplete window steps look like 'Then the following entries are available in '[string]' table:' . So I'm not able to go to definition after applying this autocomplete if step definition contains '(table1|table2)' instead of '(.*)' for this parameter.
On previous version autocomplete used to contain steps with existing parameters and it was more usefull.

Let me know if I need to explain it in more details.

**Comment by gasparnagy** [comment]
@denkorz I think these are two separate things.

1) Autocomplete popup: The new extension pups up the autocomplete dialog when you press a space after a keyword. This is to avoid annoying popups coming up during typing. BUT: *you can always force the popup* to come even in a middle of a half written step using the Ctrl+SPACE keyword combination. 

2) The step parameters are not included in the autocomplete list: This was a design decision with the new plugin for improving robustness. To be able to provide the concrete parameters, the extension would need to load & parse ALL feature files, that takes a lost of time and it is hard to cache. So therefore in the new extension we haven't implemented that feature. If you would like to check what parameters are used for a particular step definition, you can use the "Find step definition usages" command from the step definition method.

**Comment by patiqs** [comment]
This is a by design behavior. So closing this issue. Please reopen if any further assistance is needed
